staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -82,7 +81,7 @@ func NewExplainOptions(parent string, streams genericclioptions.IOStreams) *Expl
 	return &ExplainOptions{
 		IOStreams:       streams,
 		CmdParent:       parent,
-		EnableOpenAPIV3: os.Getenv("KUBECTL_EXPLAIN_OPENAPIV3") == "true",
+		EnableOpenAPIV3: cmdutil.AlphaEnabled(cmdutil.ExplainOpenapiV3Env),
```

Comment:
This is also drive-by clean up. After digging around trying to find a pattern for enabling and testing the alpha enablement env vars that we always require in kubectl and coming up empty, I decided to implement what I was looking for, and put it to use where currently relevant. Same for the test helper I added. LMK if I missed tools I should use instead (or if there are other alphas I should modify as well).